src/components/PlaidLink/index.js

```diff
@@ -0,0 +1,35 @@
+import {useCallback, useEffect} from 'react';
+import {usePlaidLink} from 'react-plaid-link';
+import {plaidLinkPropTypes, plaidLinkDefaultProps} from './plaidLinkPropTypes';
+
+const PlaidLink = (props) => {
+    const onSuccess = useCallback((publicToken, metadata) => {
+        props.onSuccess({publicToken, metadata});
+    }, []);
+
+    const {open, ready, error} = usePlaidLink({
```

Comment:
We normally should not need to use hooks (and kind of discourage it), but the Plaid library we're for React web here really only offers this option to launch the flow programmatically.